tests/generic-token-faucet.ts

```diff
@@ -1,42 +1,53 @@
 import * as anchor from "@project-serum/anchor";
 import { AnchorError, Program } from "@project-serum/anchor";
-import { getOrCreateAssociatedTokenAccount, getAccount } from "@solana/spl-token";
+import {
+  getOrCreateAssociatedTokenAccount,
+  getAccount,
+} from "@solana/spl-token";
 import { assert, expect } from "chai";
 const { SystemProgram, Transaction } = anchor.web3;
-import { DevnetTokenFaucet } from "../target/types/devnet_token_faucet";
-const { TOKEN_PROGRAM_ID, Token, ASSOCIATED_TOKEN_PROGRAM_ID } = require("@solana/spl-token");
-
-describe("devnet-token-faucet", () => {
+import { GenericTokenFaucet } from "../target/types/generic_token_faucet";
+const {
+  TOKEN_PROGRAM_ID,
+  Token,
+  ASSOCIATED_TOKEN_PROGRAM_ID,
+} = require("@solana/spl-token");
+
+describe("generic-token-faucet", () => {
   // Configure the client to use the local cluster.
-  const provider = anchor.AnchorProvider.env()
+  const provider = anchor.AnchorProvider.env();
   anchor.setProvider(provider);
 
   const payer = anchor.web3.Keypair.generate();
-  console.log("Payer Pubkey: ", payer.publicKey.toBase58())
+  console.log("Payer Pubkey: ", payer.publicKey.toBase58());
 
-  const program = anchor.workspace.DevnetTokenFaucet as Program<DevnetTokenFaucet>;
+  const program = anchor.workspace
+    .GenericTokenFaucet as Program<GenericTokenFaucet>;
 
-  let ticker = "AAA"
-  let ticker_lowercase = ticker.toLowerCase()
-  let decimals = 9
-  let amount_to_mint = 10 * 10^(decimals) 
+  let ticker = "AAA";
+  let ticker_lowercase = ticker.toLowerCase();
+  let decimals = 9;
+  let amount_to_mint = (10 * 10) ^ decimals;
```

Comment:
hold up, this is an XOR not an exponentiation right?

Reply:
Oof yeah fixed in latest commit [3253e64](https://github.com/Ellipsis-Labs/devnet-token-faucet/pull/2/commits/3253e64de2e002aa24194b326c4c5b14fd19482a)